README.md

```diff
@@ -7,8 +7,8 @@ This is an boilerplate repo to make it easy to experiment with [ES6]. It's inspi
 
 ```bash
 # Clone the repo...
-git clone https://github.com/thoughtram/es6-6to5-browserify-boilerplate.git
-cd es6-6to5-browserify-boilerplate
+git clone es6-babel-browserify-boilerplate.git
```

Comment:
Strange, the [modified version includes the fully formed URL](https://github.com/edm00se/es6-babel-browserify-boilerplate/blob/master/README.md#initial-setup).